book/website/reproducible-research/rdm/rdm-elns.md

```diff
@@ -0,0 +1,234 @@
+
+(rr-rdm-elns-choosing-lockin)=
+### Lock-in
+
+Most proprietary or open tools will permit you to export your data in one format or another.
+The quality of this export is critical to scrutinise.
+The ability to save all your lab books as PDFs is fine but if it's the only option you may loose a lot of metadata, and the ability to import your data into a new solution. 
+The ability to export your data and retain it's structure is very important to evaluate.
+
+Proprietary solutions are incentivised to attempt to lock-in their customers so that the cost of switching to a competitor is high - so be especially wary of poor export options.
+Note that this tends to change over time. 
+New providers are focused on user aquisition so data portability tends to be good at first but decline after a certain point in time.
+Firms tends to follow a pattern of shifting from a user aquisiton phase with favourable terms, to a user retention phase with less favourable ones.
+Mendeley for example began [encrypting their local database](https://www.zotero.org/support/kb/mendeley_import#mendeley_database_encryption) making it impossible to migrate your local library to different reference managers without going through their online library feature.
+This occured after they had reached a high degree of market penetration and been aquired by Elsevier.
+
+When looking for any critical piece of software for long term use the first questions to ask are:
+- Is there an Libre / open-source solution to this problem?
+- If it is a web app and/or has a server: Could I host my own fully featured instance, should I need to?
+- Is there a large/active community using the project, does it have institutional backing of some kind? 
+  This might take the form of a company which sells service contracts, or offers paid hosting ideally with feature parity with a self-hosted option. 
+  Or perhaps a foundation or other non-profit/academic organisation with robust funding.
+- What are the data export options and formats?
+
+Open solutions provide you with the assurance that if you do the appropriate preparatory work you should be able to access all of your data in it's native form.
+For example keeping copies of the ELN software that can be run offline in a VM or similar computational environment in the future.
+This is typically much easier for open applications and can be impossible for proprietary ones.
+Even if the tools are no longer maintained and in a state that can be used in production they can still be used to read the data and interact with it in (mostly) the same way.
+The data will also likely be stored in an open format from which it can relatively easily be extracted and ported to a new format.
+This may still be a lot of work but it is much more attainable than reverse engineering a proprietary format for which there is no documentation or worse - dealing with encrypted files.
```

Comment:
```suggestion
This may still be a lot of work but it is much more attainable than reverse engineering a proprietary format for which there is no documentation, or worse - dealing with encrypted files.
```